src/cpp/session/modules/SessionThemes.cpp

```diff
@@ -452,6 +452,62 @@ Error removeTheme(const json::JsonRpcRequest& request,
    return error;
 }
 
+Error getThemeName(const json::JsonRpcRequest& request,
```

Comment:
Since all this RPC does is execute R code, you might consider rewriting it using `.rs.addJsonRpcHandler` -- it'd probably be just a handful of lines in R.